common/src/main/java/org/apache/gravitino/dto/requests/RoleCreateRequest.java

```diff
@@ -79,5 +79,14 @@ public void validate() throws IllegalArgumentException {
     Preconditions.checkArgument(
         StringUtils.isNotBlank(name), "\"name\" field is required and cannot be empty");
     Preconditions.checkArgument(securableObjects != null, "\"securableObjects\" can't null ");
+    for (SecurableObjectDTO objectDTO : securableObjects) {
+      Preconditions.checkArgument(
+          StringUtils.isNotBlank(objectDTO.name()), "\" securable object name\" can't be blank");
```

Comment:
There is a space before word `securable`.

Reply:
Removed.